packages/flutter_localizations/lib/src/l10n/cupertino_pl.arb

```diff
@@ -28,10 +28,10 @@
   "cutButtonLabel": "Wytnij",
   "copyButtonLabel": "Kopiuj",
   "pasteButtonLabel": "Wklej",
-  "selectAllButtonLabel": "Zaznacz wszystko",
+  "selectAllButtonLabel": "Wybierz wszystkie",
   "tabSemanticsLabel": "Karta $tabIndex z $tabCount",
   "modalBarrierDismissLabel": "Zamknij",
   "searchTextFieldPlaceholderLabel": "Szukaj",
-  "noSpellCheckReplacementsLabel": "Nie znaleziono zastąpień",
+  "noSpellCheckReplacementsLabel": "Brak wyników zamieniania",
```

Comment:
This should be left as is, source: Apple's macOS glossary:

```
    <TextItem>
      <Description> No Replacements Found label in menu bar. Shown when the spellchecker cannot offer any guesses for a given word. </Description>
      <Position>No Replacements Found</Position>
      <TranslationSet>
        <base loc="en">No Replacements Found</base>
        <tran loc="pl">Nie znaleziono zastąpień</tran>
      </TranslationSet>
    </TextItem>
```

IMHO for Cupertino translations, the localised strings should be checked with Apple glossaries (https://developer.apple.com/download/all/?q=glossaries).

<details>
  <summary>"No Replacements Found" translations from Apple glossaries</summary>

```
% grep -r "<base loc=\"en\">No Replacements Found</base>" -A 1 | grep -v -- "^--$" | grep -v loc=\"en\" | sort
./Arabic/UIKitCore.lg-        <tran loc="ar">لم يتم العثور على بدائل</tran>
./Brazilian/UIKitCore.lg-        <tran loc="pt-BR">Nenhuma Substituição Encontrada</tran>
./Canadian French/UIKitCore.lg-        <tran loc="fr-CA">Aucune proposition</tran>
./Catalan/UIKitCore.lg-        <tran loc="ca">No s’han trobat substituts</tran>
./Croatian/UIKitCore.lg-        <tran loc="hr">Zamjene nisu nađene</tran>
./Czech/UIKitCore.lg-        <tran loc="cs">Nenalezeny žádné náhrady</tran>
./Danish/UIKitCore.lg-        <tran loc="da">Ingen erstatninger fundet</tran>
./Dutch/UIKitCore.lg-        <tran loc="nl">Geen vervanging gevonden</tran>
./Finnish/UIKitCore.lg-        <tran loc="fi">Korvaavuuksia ei löytynyt</tran>
./French/UIKitCore.lg-        <tran loc="fr">Aucune proposition</tran>
./German/UIKitCore.lg-        <tran loc="de">Kein Ersatz gefunden</tran>
./Greek/UIKitCore.lg-        <tran loc="el">Δεν βρέθηκαν αντικαταστάσεις</tran>
./Hebrew/UIKitCore.lg-        <tran loc="he">לא נמצאו חלופות</tran>
./Hindi/UIKitCore.lg-        <tran loc="hi">कोई प्रतिस्थापन सुझाव नहीं मिला</tran>
./Hong Kong/UIKitCore.lg-        <tran loc="zh-HK">找不到轉換字詞</tran>
./Hungarian/UIKitCore.lg-        <tran loc="hu">Nem található csere</tran>
./Indonesian/UIKitCore.lg-        <tran loc="id">Pengganti Tidak Ditemukan</tran>
./Italian/UIKitCore.lg-        <tran loc="it">Nessuna sostituzione trovata</tran>
./Japanese/UIKitCore.lg-        <tran loc="ja">代替候補が見つかりません</tran>
./Korean/UIKitCore.lg-        <tran loc="ko">발견된 대치 항목 없음</tran>
./Latin American/UIKitCore.lg-        <tran loc="es-419">No se encontró ninguna palabra de sustitución</tran>
./Malay/UIKitCore.lg-        <tran loc="ms">Tiada Pengganti Ditemui</tran>
./Norwegian Bokmal/UIKitCore.lg-        <tran loc="no">Fant ingen erstatninger</tran>
./Polish/UIKitCore.lg-        <tran loc="pl">Nie znaleziono zastąpień</tran>
./Portuguese/UIKitCore.lg-        <tran loc="pt-PT">Nenhuma substituição encontrada</tran>
./Romanian/UIKitCore.lg-        <tran loc="ro">Nicio înlocuire găsită</tran>
./Russian/UIKitCore.lg-        <tran loc="ru">Замен не найдено</tran>
./Simplified Chinese/UIKitCore.lg-        <tran loc="zh-CN">未找到替换文本</tran>
./Slovak/UIKitCore.lg-        <tran loc="sk">Nenašli sa žiadne náhrady</tran>
./Spanish/UIKitCore.lg-        <tran loc="es">No se ha encontrado ninguna palabra de sustitución</tran>
./Swedish/UIKitCore.lg-        <tran loc="sv">Inga ersättningar hittades</tran>
./Thai/UIKitCore.lg-        <tran loc="th">ไม่พบการแทนที่</tran>
./Traditional Chinese/UIKitCore.lg-        <tran loc="zh-TW">找不到替代文字</tran>
./Turkish/UIKitCore.lg-        <tran loc="tr">Hiçbir Değişim Bulunamadı</tran>
./Ukrainian/UIKitCore.lg-        <tran loc="uk">Замін не знайдено</tran>
./Vietnamese/UIKitCore.lg-        <tran loc="vi">Không tìm thấy mục thay thế</tran>
```
</details>

Reply:
I'll get to this with the other inconsistencies in https://github.com/flutter/flutter/issues/127871.